lib/content-services/src/lib/viewer/components/alfresco-viewer.component.ts

```diff
@@ -215,6 +215,10 @@ export class AlfrescoViewerComponent implements OnChanges, OnInit, OnDestroy {
     nodeEntry: NodeEntry;
     tracks: Track[] = [];
     readOnly: boolean = true;
+    allowedActions: { [key: string]: boolean } = {
```

Comment:
Because this property controls the visibility of some actions only on edit mode, perhaps the name could be more specific, like `allowedEditActions`? We also have other actions that are always available on readonly mode, like the zoom in/out and reset, so just "allowedActions" could be misleading.

Reply:
Done